main.py

```diff
@@ -0,0 +1,43 @@
+from cmath import log
+from fastapi import FastAPI, File, UploadFile, responses
+from deta import Deta
+import os
+
+PROJECT_KEY = os.environ['PROJECT_KEY']
+app = FastAPI()
+
+deta = Deta(PROJECT_KEY)
+drive = deta.Drive("preprocessed")
+
+
+@app.get("/")
+async def root():
+    return {"message": "Hello World"}
```

Comment:
`NIT` As above, maybe change to something more descriptive or just leave if it's going to be changed in further development

Reply:
It will be changed probably or deleted in future development, for now it's just entry point for the app to not throw 404